ToDoMate/TODO_DATA.js

```diff
@@ -0,0 +1,29 @@
+export const TODO_DATA = [
+  {
+    category: "솝트",
+    todos: ["2차 세미나 복습", "2차 과제"],
+    isDone: [true, false],
+  },
+  {
+    category: "스터디",
+    todos: ["웹심화 스터디", "JS 스터디 아티클"],
+    isDone: [true, false],
+  },
+  {
+    category: "학교",
+    todos: ["졸프 면담 준비", "최종 보고서 작성"],
+    isDone: [false, false],
+  },
```

Comment:
todos랑 isDone이 연관되어있지 않아보여서 객체를 아래처럼 정의하면 어떨까요? 
```suggestion
    {
        category: '스터디',
        todos: [
            { todoName: '웹심화 스터디', isDone: true },
            { todoName: 'JS 스터디 아티클', isDone: false },
        ],
    },
```

Reply:
저도 코드 리뷰하면서 이 생각했어요!! index값으로 맞추려고 했었는데 이 구조가 확실히 더 데이터 구조를 나타내기 좋은 것 같아요 감사합니당 ㅎㅎ